module-apps/application-alarm-clock/models/NewEditAlarmModel.hpp

```diff
@@ -48,8 +48,8 @@ namespace app::alarmClock
         void clearData() override;
         void createData();
 
-        [[nodiscard]] unsigned int getMinimalItemSpaceRequired() const override;
-        [[nodiscard]] unsigned int requestRecordsCount() override;
+        [[nodiscard]] unsigned getMinimalItemSpaceRequired() const override;
+        [[nodiscard]] unsigned requestRecordsCount() override;
         [[nodiscard]] gui::ListItem *getItem(gui::Order order) override;
         void requestRecords(uint32_t offset, uint32_t limit) override;
```

Comment:
std::uint32_t

Reply:
It cannot simply be changed to other type as these methods are derived from `ListItemProvider` interface class, so to avoid function signature mismatch I'd need to change it there too, what would imply changing it in every `ListItemProvider` implementation across the whole system.